aten/src/ATen/core/DeprecatedTypeProperties.h

```diff
@@ -34,6 +34,10 @@ class CAFFE2_API DeprecatedTypeProperties {
     return layout_from_backend(backend()) == kSparse;
   }
 
+  bool is_lazy() const {
+    return backend() == Backend::Lazy;
+  }
```

Comment:
My vote is to just not add this method to the deprecated class :>

Reply:
it's pretty much the cleanest way to get a check in autograd/engine.cpp working